spec/units/peep_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'bcrypt'
+require 'peep'
+
+describe Peep do
+  it 'creates a peep' do
+    allow(Time).to receive(:now).and_return('2019-01-01 00:00:00')
+    peep = Peep.create('My first peep', 1)
```

Comment:
I think I must be missing something here, but I'm struggling to understand how the `Peep.create` method makes use of the mocked time?